pkg/rancher-desktop/config/settingsImpl.ts

```diff
@@ -470,6 +470,10 @@ export const updateTable: Record<number, (settings: any, locked : boolean) => vo
     }
   },
   10: (settings, locked) => {
+    // virtiofs is not supported in Rancher Desktop 1.13.0
+    if (settings.experimental.virtualMachine.mount.type === MountType.VIRTIOFS) {
+      settings.experimental.virtualMachine.mount.type = MountType.REVERSE_SSHFS;
```

Comment:
This means people will need to manually switch it back _on_ afterwards. That's probably okay.